packages/transformers/webextension/src/schema.js

```diff
@@ -449,6 +448,13 @@ export const MV3Schema = ({
       additionalProperties: false,
     },
     host_permissions: arrStr,
+    sandbox: {
```

Comment:
Duplicate property?

Reply:
There are two different schemas in this file, so they are separate.